share/completions/emerge.fish

```diff
@@ -4,7 +4,7 @@
 
 function __fish_emerge_print_installed_pkgs --description 'Prints completions for installed packages on the system from /var/db/pkg'
  if test -d /var/db/pkg
-   find /var/db/pkg/ -type d | cut -d'/' -f5-6 | sort | uniq | \
+   find /var/db/pkg/ -type d | cut -d'/' -f5-6 | sort -u | \
```

Comment:
[Yes, it is](http://pubs.opengroup.org/onlinepubs/9699919799/utilities/sort.html). Okay then.